aerialist/px4/k8s_agent.py

```diff
@@ -96,7 +99,7 @@ def run(self):
                 logger.debug("files downloaded")
 
             for test_log in os.listdir(local_folder):
-                if test_log.endswith(".ulg") and (
+                if ".ulg" in test_log and (
```

Comment:
Since we are zipping the file, we longer have file name ending with .ulg

Reply:
let's keep the .ulg file as is. don't zip it